qiskit/qobj/_qobj.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Models for QObj and its related components."""
+
+from types import SimpleNamespace
+
+from ._utils import QObjType
+
+
+class QObjStructure(SimpleNamespace):
```

Comment:
To be honest, I don't like the name but I'm not able to provide anything better. What about `QObjEntry` or `QObjItem`? Non-blocking, anyway.

Reply:
Updated!